web/viewer.html

```diff
@@ -686,6 +686,18 @@
 <!--#endif-->
       </div>  <!-- dialogContainer -->
 
+      <div id="toastContainer">
+        <div id="annotationRemovedToast" class="messageBar" hidden>
+          <div>
+            <div>
+              <span class="description" data-l10n-id="pdfjs-editor-messagebar-annotation-removed-text" data-l10n-args='{"type": "annotation"}'>[Annotation] removed</span>
+            </div>
+            <button id="annotationRemovedUndoButton" class="undoButton" type="button" tabindex="0" title="Undo"><span data-l10n-id="pdfjs-editor-messagebar-annotation-undo-button">Undo</span></button>
+            <button id="annotationRemovedCloseButton" class="closeButton" type="button" tabindex="1" title="Close"><span data-l10n-id="pdfjs-editor-messagebar-annotation-close-button">Close</span></button>
```

Comment:
Don't use `tabindex=X` with `X > 0`.

Reply:
Thanks for the pointer, I was totally unsure of how `tabIndex` works to begin with :sweat_smile: 

I've switched it with `0`, hope that's what was needed. I tested and the button is focusable.